src/backend/downloadmanager/downloadqueue.ts

```diff
@@ -142,7 +144,7 @@ async function addToQueue(element: DMQueueElement) {
       'dependencies' in installInfo.manifest
     ) {
       const newDependencies = installInfo.manifest.dependencies
-      if (newDependencies?.length) {
+      if (newDependencies?.length || !existsSync(gogRedistPath)) {
```

Comment:
Looking over this again shouldn't this be an and?

Reply:
we are about to create that path, it's a check to make sure it's not removed and we always have ISI downloaded

Reply:
I might need to add a fallback to newDependencies in case it's undefined